init_pagelinks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+which docker
+which mycli
+echo "INITALIZING MARIADB"
+docker run -p3306:3306 \
+    --name pagelinks-theodorc-eivinkop \
+    -e MARIADB_ROOT_PASSWORD=my-secret-pw \
```

Comment:
```suggestion
    -e MARIADB_ROOT_PASSWORD="${MARIADB_ROOT_PASSWORD}" \
```
haha vet ikke om denne henter ut fra .env?